Sources/HAP/Server/Device.swift

```diff
@@ -48,9 +50,92 @@ public class Device {
     public let setupCode: String
     let storage: Storage
     let pairings: Pairings
-    public let accessories: [Accessory]
+    public var accessories: [Accessory]
     internal var characteristicEventListeners: [Box<Characteristic>: WeakObjectSet<Server.Connection>]
     public var onIdentify: [(Accessory?) -> Void] = []
+    internal var configuration: Configuration
+    public var onConfigurationChange: [(Device) -> Void] = []
+    public let isBridge: Bool
```

Comment:
We could change this into a computed property:

```swift
public let isBridge: Bool {
    return accessories[0].type == .bridge
}
```

Reply:
I purposely didn't make it a computed property for efficiency reasons. Once the test has been made at initialisation, there is no need to recomputed it, as the instance can never change type.

Reply:
Ok